ruby/arguments.rb

```diff
@@ -0,0 +1,10 @@
+"""Script to read an argument and print as lowercase in stdout.
+Note that command line arguments are not passed from stdin"""
```

Comment:
I left this note for me to remember that args aren't the same as input/stdin. Is this true as written? I'm forgetting exactly how that works

Reply:
> Note that command line arguments are not passed from stdin

I don't understand what this is supposed to convey.

- command line arguments are given to a new process as an array of strings (I believe the "V" in "ARGV" stands for "Vector")
- stdin is a file that is given to a new process (and sometimes the file is empty)

Reply:
Ok thanks. That's a good breakdown. I guess I don't need to include a note, but I kept forgetting the difference between the two.